scripts/setup-operators.sh

```diff
@@ -0,0 +1,175 @@
+install_sealed_secrets_operator(){
+oc create -f - <<EOF
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  generateName: cicd-
+  namespace: cicd
+spec:
+  targetNamespaces:
+  - cicd
+EOF
+
+oc create -f - <<EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: sealed-secrets-operator-helm
+  namespace: cicd
+spec:
+  channel: alpha
+  name: sealed-secrets-operator-helm
+  source: community-operators
+  sourceNamespace: openshift-marketplace
+EOF
+}
+
+install_sealed_secrets_operator
+# sealed secrets operator status check
+count=0
+while [ "$count" -lt "5" ];
+do
+    operator_status=`oc get csv -n cicd | grep sealed-secrets-operator`
+    if [[ $operator_status == *"Succeeded"* ]]; then
+        break
+    else
+        count=`expr $count + 1`
+        sleep 10
+    fi
+done
+echo "Completes sealed secrets operator installation"
+
+echo "Starting sealed secrets operator instance creation"
+create_sealed_secrets_operator_instance(){
+oc create -f - <<EOF
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecretController
+metadata: 
+  name: sealedsecretsontroller
+  namespace: cicd
+spec: 
+  affinity: {}
+  controller: 
+    create: true
+  crd: 
+    create: true
+    keep: true
+  image: 
+    pullPolicy: IfNotPresent
+    repository: "quay.io/bitnami/sealed-secrets-controller@sha256:8e9a37bb2e1a6f3a8bee949e3af0e9dab0d7dca618f1a63048dc541b5d554985"
+  ingress: 
+    annotations: {}
+    enabled: false
+    hosts: 
+      - chart-example.local
+    path: /v1/cert.pem
+    tls: []
+  networkPolicy: false
+  nodeSelector: {}
+  podAnnotations: {}
+  podLabels: {}
+  priorityClassName: ""
+  rbac: 
+    create: true
+    pspEnabled: false
+  resources: {}
+  secretName: sealed-secrets-key
+  securityContext: 
+    fsGroup: ""
+    runAsUser: ""
+  serviceAccount: 
+    create: true
+    name: ""
+  tolerations: []
+EOF
+}
+create_sealed_secrets_operator_instance
+echo "Completes sealed secrets operator instance creation"
+
+echo "Starting openshift pipelines operator installation"
+install_openshift_pipelines_operator() {
+oc create -f - <<EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-pipelines-operator-rh
+  namespace: openshift-operators
+spec:
+  channel: ocp-4.5
+  name: openshift-pipelines-operator-rh
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+EOF
+}
+
+install_openshift_pipelines_operator
+# pipelines operator status
+count=0
+while [ "$count" -lt "5" ];
+do
+    operator_status=`oc get csv -n openshift-operators | grep openshift-pipelines-operator`
+    if [[ $operator_status == *"Succeeded"* ]]; then
+        break
+    else
+        count=`expr $count + 1`
+        sleep 10
+    fi
+done
+echo "Completes openshift pipelines operator installation"
+
+echo "Install the Buildah ClusterTask"
+oc replace -f https://raw.githubusercontent.com/redhat-developer/kam/master/docs/updates/buildah.yaml
```

Comment:
How about a comment about why we need this (which might help someone determine whether or not it's safe to remove?)

Reply:
https://github.com/redhat-developer/kam/pull/54/files#diff-2da0bb63674f91e041a1df77783b9cb04842a32916a2307e525a00d4be8c22bfR103